### ISSUE TITLE
Plugin execution not covered by lifecycle configuration

### ISSUE THREAD

**Issue by dbwiddis** [dblock/oshi#52]
My pom.xml is complaining about this execution:
```xml
            <plugin>
                <groupId>com.mycila</groupId>
                <artifactId>license-maven-plugin</artifactId>
                <executions>
                    <execution>
                        <phase>compile</phase>
                        <goals>
                            <goal>format</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
```
Apparently there's something amiss with the `compile` phase.  Eclipse helpfully offers me to "permanently mark goal format ... as ignored" but something tells me that's not the right thing to do.  

According to [the M2Eclipse docs](http://www.eclipse.org/m2e/documentation/m2e-execution-not-covered.html) "Project build lifecycle mapping can be configured in a project’s `pom.xml`, contributed by Eclipse plugins, or defaulted to the commonly used Maven plugins shipped with m2e. We call these “lifecycle mapping metadata sources”. m2e will create error marker like below for all plugin executions that do not have lifecycle mapping in any of the mapping metadata sources."

So clearly I don't have the appropriate lifecycle mapping ... but shouldn't that lifecycle mapping be included in the `pom.xml`?

While searching for an answer, I found [this StackOverflow answer](http://stackoverflow.com/questions/6352208/how-to-solve-plugin-execution-not-covered-by-lifecycle-configuration-for-sprin) which suggests wrapping the plugins within `<pluginManagement>` tags... apparently only the first half of the plugins are within those tags, there's a whole second group outside of the tags.  Moving the closing tag (and deleting the duplicate close/open `<plugins>` tags) appears to resolve this issue.  

But I'm just guessing here.  Can @hazendaz please take a look?

**Comment by hazendaz** [comment]
M2e life cycle issues can safely be ignored.  I've got a few already setup in the pom but missed this one as I have m2e plugin for it.  I can fix that tomorrow so you stop seeing the error.  It's related only to eclipse.  That way everyone gets same result without extra work.

--- Original Message ---

**Comment by hazendaz** [comment]
Fixed on #55 I had m2e connector plugin in eclipse.  I removed that so I could see the same error, then I added this to the set of execution filters we already have in place for eclipse.  It otherwise had no impact on the build.  And now we have better portability :)
